spec/definitions/TokenRenewV0.yaml

```diff
@@ -1,28 +1,55 @@
 type: object
 properties:
+  id:
+    title: Token id
+    description: Id of the token
+    type: string
+    format: integer
+    readOnly: true
   user:
     title: User id
     description: Id of the user resource
     type: string
+    readOnly: true
   enterprise:
     title: Enterprise id
     description: Id of the enterprise resource
     type: string
     format: uuid
+    readOnly: true
   token:
     title: API Token
     description: API token for accessing esper APIs
     type: string
+    readOnly: true
```

Comment:
why is this field read only?

Reply:
Will change.

Reply:
@abdulMuneerM  I think read only should be fine. readonly means api user will not be able to update any of these parameters.

Reply:
Ok